src/com/fonts.less

```diff
@@ -8,7 +8,7 @@
 html, button, input, ._font1 {
 	/* Lato - Standard text font (Google Fonts) */
 	font-family: "Roboto", "Helvetica Neue", "Arial Nova", "Segoe UI", "Ubuntu Light", sans-serif;
-/*	font-family: "Lato", "Helvetica Neue", "Roboto", "Arial Nova", "Segoe UI", "Ubuntu Light", sans-serif;*/
+	/*	font-family: "Lato", "Helvetica Neue", "Roboto", "Arial Nova", "Segoe UI", "Ubuntu Light", sans-serif;*/
```

Comment:
why?

Reply:
The Less linter wants the comment to be indented correctly within it's scope.

Reply:
Okay.